src/components/sbb-header/sbb-header.scss

```
@@ -43,7 +44,7 @@
   z-index: 10;
   transition-property: var(--sbb-header-transition-property);
   transition-duration: var(--sbb-header-animation-duration);
```

maybe also rename the --sbb-header-animation-duration 
```suggestion
  transition-duration: var(--sbb-header-transition-duration);
```